garrysmod/gamemodes/terrortown/gamemode/corpse.lua

```diff
@@ -36,6 +36,12 @@ end
 -- If detective mode, announce when someone's body is found
 local bodyfound = CreateConVar("ttt_announce_body_found", "1")
 
+-- opportunity for addons to introduce ways for traitors(ideally) to interrupt corpse searching.
+function GM:TTTInterruptCorpseIdentify(identifier,corpse,corpse_is_traitor)

```

Comment:
Also, I would suggest `TTTCanIdentifyCorpse` and `TTTCanSearchCorpse` as hook names, with the default implementation returning true. This fits in a bit better with standard hooks like PlayerCanPickupWeapon.